### PR TITLE
Line Items: For small desktop

### PR DESCRIPTION
**Pull request by daniel-samson**
## Description

This change comes from feedback from the users. I have made the line items fit into a small desktop (1024x768). This is achieved by making the elements and text smaller. Please bear in mind than on a small screen they look correct. I have added a break point for large desktops too which has bigger text and elements.

screenshot_20170117_112332

For tablet (vertical), we need to come up with a solution to save screen space.

## Motivation and Context

* Makes line items usable for SuiteP users who have smaller displays.

## How To Test This

* Test Invoices
* Test Quotes

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/AOS_Products_Quotes/line_items.js [hunk 1]
```diff
@@ -551,32 +551,32 @@ function insertGroup()
   footer_cell.innerHTML="<input type='button' tabindex='116' class='button add_product_line' value='"+SUGAR.language.get(module_sugar_grp1, 'LBL_ADD_PRODUCT_LINE')+"' id='"+productTable.id+"addProductLine' onclick='insertProductLine(\""+productTable.id+"\",\""+groupn+"\")' />";
   footer_cell.innerHTML+=" <input type='button' tabindex='116' class='button add_service_line' value='"+SUGAR.language.get(module_sugar_grp1, 'LBL_ADD_SERVICE_LINE')+"' id='"+serviceTable.id+"addServiceLine' onclick='insertServiceLine(\""+serviceTable.id+"\",\""+groupn+"\")' />";
   if(enable_groups){
-    footer_cell.innerHTML+="<span class='totals'>"+SUGAR.language.get(module_sugar_grp1, 'LBL_TOTAL_AMT')+":&nbsp;&nbsp;<input name='group_total_amt[]' id='"+ table.id +"total_amt' class='group_total_amt' maxlength='26' value='' title='' tabindex='120' type='text' readonly></span>";
+    footer_cell.innerHTML+="<nput class='totals'><label>"+SUGAR.language.get(module_sugar_grp1, 'LBL_TOTAL_AMT')+":</label><input name='group_total_amt[]' id='"+ table.id +"total_amt' class='group_total_amt' maxlength='26' value='' title='' tabindex='120' type='text' readonly></span>";
```
Comment (mattlorimer) [inline modules/AOS_Products_Quotes/line_items.js]: nput

File: modules/AOS_Products_Quotes/line_items.js [hunk 2]
```diff
@@ -551,32 +551,32 @@ function insertGroup()
...
+    footer_cell.innerHTML+="<span class='totals'><label>"+SUGAR.language.get(module_sugar_grp1, 'LBL_TOTAL_AMT')+":</label><input name='group_total_amt[]' id='"+ table.id +"total_amt' class='group_total_amt' maxlength='26' value='' title='' tabindex='120' type='text' readonly></span>";
 
     var footer_row2=tablefooter.insertRow(-1);
     var footer_cell2 = footer_row2.insertCell(0);
     footer_cell2.scope="row";
     footer_cell2.colSpan="20";
-    footer_cell2.innerHTML="<span class='totals'>"+SUGAR.language.get(module_sugar_grp1, 'LBL_DISCOUNT_AMOUNT')+":&nbsp;&nbsp;<input name='group_discount_amount[]' id='"+ table.id +"discount_amount' class='group_discount_amount' maxlength='26' value='' title='' tabindex='120' type='text' readonly></span>";
+    footer_cell2.innerHTML="<span class='totals'><label>"+SUGAR.language.get(module_sugar_grp1, 'LBL_DISCOUNT_AMOUNT')+":</label></label><input name='group_discount_amount[]' id='"+ table.id +"discount_amount' class='group_discount_amount' maxlength='26' value='' title='' tabindex='120' type='text' readonly></label>";
```
Comment (samus-aran) [inline modules/AOS_Products_Quotes/line_items.js]: Should there be two label tags?

File: themes/SuiteP/css/editview.scss [hunk 3]
```diff
@@ -550,11 +556,11 @@ table.view {
 
 #lineItems {
   display: inline;
-  font-size: 14px;
+  font-size: 11px;
   font-weight: 700;
   color: $main-font-color;
   padding: 0 0 0 8px;
-  line-height: 48px;
+  //line-height: 48px;
```
Comment (samus-aran) [inline themes/SuiteP/css/editview.scss]: Remove

Comment (daniel-samson) [inline themes/SuiteP/css/editview.scss]: Updated with fixes.

**Pull request merged**